src/runtime/object.h

```diff
@@ -99,126 +99,126 @@ inline void cnstr_set_float(b_obj_arg o, unsigned offset, double v) { lean_ctor_
 // =======================================
 // Closures
 
-void free_closure_obj(object * o);
```

Comment:
The declaration `free_closure_obj` was removed because the function doesn't exist.